dpe/src/commands/rotate_context.rs

```diff
@@ -22,9 +22,8 @@ bitflags! {
 #[derive(Debug, PartialEq, Eq, zerocopy::FromBytes)]
 #[cfg_attr(test, derive(zerocopy::AsBytes))]
 pub struct RotateCtxCmd {
-    handle: ContextHandle,
-    flags: RotateCtxFlags,
-    target_locality: u32,
```

Comment:
Why was `target_locality` removed?

Reply:
Looks like we don't use it anywhere in rotate context. Also not in the spec

Reply:
Hmm. The upstream spec supports it, but we could disallow it in the profile. I think it's preferred from a security perspective if handles can't be moved between localities.

So fine to leave this as-is. We just need to update the profile spec to say that the `target_locality` parameter is not supported and that handles will always be rotated into the same locality.